superset/db_engine_specs/base.py

```diff
@@ -954,12 +952,12 @@ def select_star(  # pylint: disable=too-many-arguments,too-many-locals
 
         :param database: Database instance
         :param table_name: Table name, unquoted
-        :param engine: SqlALchemy Engine instance
+        :param engine: SqlAlchemy Engine instance
```

Comment:
nit: while we're fixing typos, let's get it _completely_ right 😆 
```suggestion
        :param engine: SQLAlchemy Engine instance
```